### ISSUE TITLE
MODS warnings passed

### ISSUE THREAD

**Issue by atz** [sul-dlss/stanford-mods#25]
```
/Users/atz/.rvm/gems/ruby-2.1.2/gems/mods-2.0.1/lib/mods/nom_terminology.rb:9: warning: already initialized constant Mods::LANG_ATTRIBS
/Users/atz/.rvm/gems/ruby-2.1.2/gems/mods-2.0.1/lib/mods/nom_terminology.rb:5: warning: previous definition of LANG_ATTRIBS was here
```

Since the warnings are from the same file, that suggests it might be mod's problem alone.

**Comment by atz** [comment]
Fixed this in mods, will be resolved here by updated dep.

**Comment by atz** [comment]
Resolved in `v1.1.3`
